src/commands/app/db/getCollectionNames.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
suggestion:

should we merge this with getCollectionInfos and add a flag --short/--only-names?

Reply:
relates to the second general comment